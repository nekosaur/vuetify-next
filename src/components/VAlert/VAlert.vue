<template>
  <div class="foo">this is an alert</div>
</template>

<script lang="ts">
  import { defineComponent } from 'vue'
  import './VAlert.scss'

  export default defineComponent({
    name: 'VAlert',
    setup () {
      return {}
    },
  })
</script>
