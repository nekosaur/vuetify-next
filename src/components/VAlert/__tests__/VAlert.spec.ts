import { mount } from '@vue/test-utils'
import puppeteer from 'puppeteer'
import VAlert from '../VAlert.vue'

describe('VAlert', () => {
  test('uses mounts', async () => {
    const wrapper = mount(VAlert)
    expect(wrapper.html()).toContain('this is an alert')
  })

  test('snapshot', async () => {
    const wrapper = mount(VAlert)
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('image snapshot', async () => {
    const browser = await puppeteer.launch()
    const page = await browser.newPage()
    await page.goto('http://localhost:3000/dev')
    const image = await page.screenshot()
    expect(image).toMatchImageSnapshot()
    await browser.close()
  })
})
