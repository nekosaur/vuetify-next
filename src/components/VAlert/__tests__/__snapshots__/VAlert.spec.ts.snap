// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VAlert snapshot 1`] = `"<div class=\\"foo\\">this is an alert</div>"`;
